api_app/views.py

```diff
@@ -77,12 +80,32 @@ def _multi_analysis_request(
     logger.info(f"New Jobs added to queue <- {repr(jobs)}.")
 
     # Check if task is test or not
-    if settings.STAGE_CI:
+    if settings.STAGE_CI and not settings.FORCE_SCHEDULE_JOBS:
```

Comment:
Instead of having another variable and to run a job that you don't actually need to run (because, correct me if I'm wrong, but you want in the CI to verify the configuration), i think is a better solution to **always** do the configuration in the tests (hoping that this will not break other tests) and, `if not settings.STAGE_CI`, run the celery task.
In summary, I would remove this new settings variable, that we have already too many, and move the check CI/not CI before calling `celery_app.send_task`

Reply:
I need the actual `celery_app.send_task` to be executed, too, as only then would my `before_task` get triggered. So, stopping the call would prevent the test from running

Reply:
Ok, read the construct that you made, and even if I would have preferred to using the mockup package to mock `start_analyzer` and then verify that it was called with the correct args/kwargs, i do not actually dislike your solution. I think is very creative and imho we could adopt the approach even in other parts of the test suite to verify that the celery tasks have the correct arguments.

Reply:
Thanks a lot :)